modules/Movecraft/src/main/resources/types/Airship.craft

```diff
@@ -0,0 +1,141 @@
+name: Airship
+maxSize: 5000
+minSize: 1000
+allowedBlocks: 
+    - "#planks"
+    - water
+    - lava
+    - "#logs"
+    - "#glass"
+    - lapis_block
+    - dispenser
+    - note_block
+    - "#beds"
+    - sticky_piston
+    - piston
+    - piston_head
+    - "#wool"
+    - gold_block
+    - iron_block
+    - "#slabs"
+    - bricks
+    - bookshelf
+    - torch
+    - fire
+    - "#stairs"
+    - "#chests"
+    - redstone_wire
+    - diamond_block
+    - crafting_table
+    - furnace
+    - blast_furnace
+    - smoker
+    - "#signs"
+    - "#doors"
+    - ladder
+    - lever
+    - "#pressure_plates"
+    - redstone_torch
+    - redstone_wall_torch
+    - "#buttons"
+    - clay
+    - "#fences"
+    - netherrack
+    - glowstone
+    - cake
+    - repeater
+    - "#trapdoors"
+    - "#stone_bricks"
+    - iron_bars
+    - "#fence_gates"
+    - nether_brick
+    - enchanting_table
+    - brewing_stand
+    - cauldron
+    - end_stone
+    #- dagon_egg does not work for some reason???
+    - redstone_lamp
+    - ender_chest
+    - tripwire_hook
+    - tripwire
+    - emerald_block
+    - beacon
+    - "#walls"
+    - flower_pot
+    - "#heads"
+    - "#anvil"
+    - comparator
+    - daylight_detector
+    - redstone_block
+    - hopper
+    - quartz_block
+    - chiseled_quartz_block
+    - quartz_pillar
+    - smooth_quartz
+    - "#rails"
+    - dropper
+    - "#terracotta"
+    - "#concrete"
+    - slime_block
+    - "#carpets"
+    - coal_block
+    - end_stone_bricks
+    - smooth_stone
+    - bell
+    - cartography_table
+    - composter
+    - fletching_table
+    - grindstone
+    - lantern
+    - lectern
+    - loom
+    - smithing_table
+    - stonecutter
+
+forbiddenBlocks:
+    - piston_head
+
+canCruise: true
+rotateAtMidpoint: true
+canFly: true
+maxHeightLimit: 135
+maxHeightAboveGround: 50
+vertCruiseSkipBlocks: 3
```

Comment:
I understand that this is being done to bring it back in line with Airship Pirates mechanics but I'm unsure if we want the craft files changed like that

Reply:
I also failed to mention that BigAirship CruiseSkipBlocks have also been changed. I suppose it's up to the rest of the team to decide whether these should be changed.